src/main/kotlin/org/jetbrains/fortran/lang/lexer/FortranLexer.flex

```diff
@@ -125,6 +127,19 @@ CPPCOMMENT="#"\040*"if"\040*0({EOL}[^\r\n]*)*{EOL}"#"\040*"endif"{EOL}
 }
 
 
+<YYINITIAL, FREEFORM_LINE_CONTINUE, FREEFORM, FIXEDFORM, FREE_FORMAT_STR, FIXED_FORMAT_STR>{
```

Comment:
What was the point of this changes?
I've pushed them into master today, but now I see that it was a mistake because respecting macros only in first column leads to parsing error when macro is tabulated for some reason. That shouldn't happen

Reply:
By default fortran preprocessor runs in "traditional" mode. Unlike C preprocessor it respects directives only at first column. I agree, parsing might be not perfect place to handle it, but it must be taken into account.